### PR TITLE
Test run notes 2022 01 19

### PR DESCRIPTION
**Pull request by hawstom**
I hope this is in the spirit of what you expected. I avoided any style changes.

### CODE REVIEW

File: Min-Max_Guide.md [hunk 1]
```diff
@@ -118,7 +118,7 @@ Eventually, you will stop worrying about this and will only eat from a large sta
 
 ### Energy Recovery Mechanics
 
-- On the day after a player has gained one or more skill level increases, they will always start with full energy, even if they passed out on the previous day. They will also still start with full energy even if they were below 0 energy and had the "exhausted" icon showing above the energy bar.
+- On the day after a player has gained one or more sA level increases, they will always start with full energy, even if they passed out on the previous day. They will also still start with full energy even if they were below 0 energy and had the "exhausted" icon showing above the energy bar.
```
Comment (Zamiell) [inline Min-Max_Guide.md:121]: ?

File: Min-Max_Guide.md [hunk 2]
```diff
@@ -272,72 +272,71 @@ The plan for the day is to meet everybody in the town and save money for tomorro
 An alternate strategy for Spring 1 would be to ignore all of the NPCs and focus on killing every weed on the map + every weed on the farm, which has the advantage of getting a few more mixed seeds planted on the first day. However, in the "meet everybody" route, you still kill plenty of weeds, making the overall number of mixed seeds very similar. Thus, the marginal number of mixed seeds gathered in the alternate route is not worth more than the early 100g.
 
 - Take the Parsnip Seeds.
-- Immediately chop down trees until foraging level 1. We want to get level 1 before we leave to go foraging so that we have a chance to get silver-quality Spring Onions and such. (Focus on **only** chopping down trees. The route is very tight.)
+- Immediately top 9 (8 1/3 for) trees until foraging level 1 and consequently clear minimal weeds starting along your path south out of your farm and ending near your house. We want to get level 1 before we leave to go foraging so that we have a chance to get silver-quality Spring Onions and such. (Focus on **only** chopping down trees. The route is very tight.)
```
Comment (Zamiell) [inline Min-Max_Guide.md:275]: ?

File: Min-Max_Guide.md [hunk 3]
```diff
@@ -272,72 +272,71 @@ The plan for the day is to meet everybody in the town and save money for tomorro
...
   - For the same reason, do not axe/hoe any seeds in the ground until Spring 2, or they will be destroyed.
   - Do not chop the stumps; only chop a tree until it falls, then move on to the next one. (It yields more wood and more foraging XP to do it this way.)
   - As soon as a tree starts to fall, walk towards where the wood will spawn and immediately pause the game. This prevents in-game time from passing, but the tree falling animation will still complete. Additionally, all of the wood will automatically be collected by the player while the game is still paused. (You should use this strategy whenever you are chopping trees and not bothering to kill the stumps.)
 - Once foraging level 1, craft a chest, place it next to the house, and empty all inventory except for scythe + axe + pickaxe + hoe.
 - Exit south to the forest. (The time should be around 9:20 AM.)
-- Head left past the Traveling Cart area, down to the Abandoned House, right to the Spring Onion area, and up to Marnie's Ranch.
+- Head west past the Traveling Cart area, south to the Abandoned House, east to the Spring Onion area, and north to Marnie's Ranch.
```
Comment (Zamiell) [inline Min-Max_Guide.md:282]: the map doesn't necessarily correspond to the compass cardinal directions, so adding them would be a bug

File: Min-Max_Guide.md (hunk 1 above)
Comment (hawstom) [reply Min-Max_Guide.md:121]: Oops. Will fix.

File: Min-Max_Guide.md (hunk 3 above)
Comment (hawstom) [reply Min-Max_Guide.md:282]: I intended not to do this until I asked about the style convention on that. In the future I hope to proofread for the opposite, then, if that is the style. I will fix.

File: Min-Max_Guide.md (hunk 2 above)
Comment (hawstom) [reply Min-Max_Guide.md:275]: Will try to fix.

File: Min-Max_Guide.md [hunk 4]
```diff
@@ -282,62 +282,61 @@ An alternate strategy for Spring 1 would be to ignore all of the NPCs and focus
 - Head left past the Traveling Cart area, down to the Abandoned House, right to the Spring Onion area, and up to Marnie's Ranch.
   - Pick up all foragable items along the way.
   - Try to kill as many weeds as possible, but do not go too far out of your way for just 1 weed (since you need to make it to Jas at 11:20 AM).
-- Meet Jas as she exits her house around 11:20 AM.
-- Meet Haley as she walks towards Leah's house.
+- Meet Jas as she walks from her house to her swing starting at 11:20 AM.
+- Meet Haley as she walks towards Leah's house passing it around 11:40 AM.
 - Go to town.
-- Check the two garbage cans next to Jodi's house + Haley's house.
+- Check the two garbage cans next to Jodi's house + Haley's house and clear weeds west of .
```
Comment (Zamiell) [inline Min-Max_Guide.md:288]: ?

**Comment by Zamiell** [comment]
can you change all instances of south, east, west, etc to directions?

**Comment by hawstom** [comment]
Yeah. I am hoping to do that to the entire guide in a separate branch.
--
Tom Haws 480-201-5476 (or NPS 480-389-4583)
"Forgiveness is the highest and most beautiful form of love. In return, you
will receive untold peace and happiness." -Dr. Robert Muller

**Comment by Zamiell** [comment]
ok then i'll merge this for now, thanks

**Pull request merged**